Userland/Libraries/LibCore/Socket.h

```diff
@@ -248,7 +248,7 @@ class UDPSocket final : public Socket {
             // With UDP datagrams, reading a datagram into a buffer that's
             // smaller than the datagram's size will cause the rest of the
             // datagram to be discarded. That's not very nice, so let's bail
-            // early, telling the caller that he should allocate a bigger
+            // early, telling the caller that they should allocate a bigger
```

Comment:
Both would be presumably fine, the current comment style usually is targeted at the person who wrote the code instead of the code itself (as we make the person aware that they need to consider this possibility).